test/sanity/app-getproxyforurl/index.html

```diff
@@ -15,6 +15,14 @@
 	} else {
 		document.write('<h1 id="result">failure: expect "' + expect + '" but get ' + proxy +'</h1>');
 	}
+        nw.App.setProxyConfig('dummy:8080', 'http://foopy/proxy.pac');
+        var proxy = nw.App.getProxyForURL('http://www.example.com/');
+        var expect = 'DIRECT';
+        if (proxy === expect) {
+                document.write('<h1 id="result">success</h1>');
+        } else {
+                document.write('<h1 id="result">failure: expect "' + expect + '" but get ' + proxy +'</h1>');
+        }
 	</script>
```

Comment:
http://docs.nwjs.io/en/latest/For%20Developers/Writing%20Test%20Cases%20for%20NW.js/